### PR TITLE
New Items: Short and long leggings

### PR DESCRIPTION
**Pull request by TheGnarp**
Added two type of tight leggings and decided to make use of the new multi-color feature too.

### CODE REVIEW

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 1]
```diff
@@ -262,6 +262,18 @@ var AssetFemale3DCG = [
...
 			{ Name: "JeansShorts", Value: 20, Priority: 22, Hide: ["ItemVulvaPiercings"], HideItem: ["ItemButtAnalBeads2", "ItemVulvaHempRopeBelt", "ItemVulvaVibratingDildo", "ItemVulvaInflatableVibeDildo", "ItemVulvaClitSuctionCup", "ItemVulvaHeavyWeightClamp", "ItemVulvaClitAndDildoVibratorbelt", "ItemVulvaShockDildo"] },
+			{ 
+				Name: "Leggings1", Value: 15, Priority: 21, DefaultColor: "#4499c4", Hide: ["ItemVulvaPiercings", "Panties"], HideItem: ["ItemButtAnalBeads2", "SocksSocksFur", "SocksSocks4", "SocksSocks5", "SocksSocks6", "VibratingLatexPanties", "VibratingDildo", "InflatableVibeDildo", "ClitSuctionCup", "TapeStrips", "BenWaBalls", "HeavyWeightClamp", "ItemVulvaPiercingsVibeHeartClitPiercing", "ItemVulvaShockDildo"], Layer: [
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:266]: The hide items for both have some invalid entries that do not match the <GroupName><AssetName> pattern. Can you double check which one needs to be kept and fixed?

**Review by ace-1331 — CHANGES_REQUESTED** [review]
Just 1 minor comment on this one, everything else looks good

File: BondageClub/Assets/Female3DCG/Female3DCG.js (hunk 1 above)
Comment (TheGnarp) [reply BondageClub/Assets/Female3DCG/Female3DCG.js]: Forgot prefixes for some items. Should be fixed now and I tested if they actually vanish.

**Review by ace-1331 — APPROVED** [review]

**Review by elliesec — APPROVED** [review]
They look lovely!

**Pull request merged**